client/CHANGELOG.md

```diff
@@ -2,9 +2,9 @@
 
 All notable changes to this project will be documented in this file.
 
-## [0.4.5] - 2021-05-05
+## [0.4.6] - 2021-05-05
 
-- Added the export of `generateJWE`
+- Added the export of `generateJWE`.
```

Comment:
```suggestion
- Added the export of `generateJWE`. This was missing from `0.4.4`
```
Otherwise it could raise questions. This way we give all the need information about why this change is needed.